<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModality::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>257</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CrappyCounter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="lbl_number">
      <property name="font">
       <font>
        <pointsize>50</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: green</string>
      </property>
      <property name="text">
       <string>42</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnInc">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>+1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnDec">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>-1</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
